docs/framework.md

```diff
@@ -42,10 +42,13 @@ if ($pharPath) {
     }
 }
 
-$commandClasses = [ \MyProject\Commands\RoboCommands::class ];
-$runner = new \Robo\Runner($commandClasses);
-$output = new \Symfony\Component\Console\Output\ConsoleOutput();
-$statusCode = $runner->execute($_SERVER['argv'], $output, 'MyAppName', '0.0.0-alpha0');
+$commandClasses = [ \MyProject\Commands\RoboFile::class ];
+$statusCode = Robo::run(
+    $_SERVER['argv'], 
+    $commandClasses, 
+    'MyAppName', 
+    '0.0.0-alpha0'
+);
```

Comment:
I like it, documentation is simplified. However there should be a way to disable default `\Symfony\Component\Console\Output\ConsoleOutput` and to replace it with something else. This should be mentioned in docs